src/components/MenuContent/MenuContent.tsx

```diff
@@ -0,0 +1,85 @@
+import React, { ComponentType } from 'react';
+import { menuContent } from 'cms-content/footer';
+import FeaturedSection from './FeaturedSection';
+import LearnSection from './LearnSection';
+import AboutUsSection from './AboutUsSection';
+import { ContentWrapper } from './Menu.style';
+import { useBreakpoint } from 'common/hooks';
+import {
+  Experiment,
+  ExperimentID,
+  Variant,
+  VariantID,
+} from 'components/Experiment';
+import { FeaturedItem, SectionId } from 'cms-content/footer';
+
+const FeaturedSectionVariant: React.FC<{
+  featuredSections: FeaturedItem[];
+  onClick: (label: string) => void;
+}> = ({ featuredSections, onClick }) => {
+  const apiLink = featuredSections.filter(
+    section => section.iconId === SectionId.API,
+  );
+  const dailyDownloadLink = featuredSections.filter(
+    section => section.iconId === SectionId.DAILY_DOWNLOAD,
+  );
+  const alertsLink = featuredSections.filter(
+    section => section.iconId === SectionId.ALERTS,
+  );
```

Comment:
Maybe using `find` would be better, that way you can just add the items to the array without the `...`

```ts
const apiLink = featuredSections.find(
  ({ iconId }) => iconId === SectionId.API,
);

const orderedSectionsA = [apiLink, alertsLink, dailyDownloadLink];
```

creating a map by Id might be more compact too

```ts
const sectionById = keyBy(featuredSections, section => section.iconId);

const orderedSectionsA = [
  sectionsById[SectionId.API],
  // ...
]
```